UI/auth-youtube.cpp

```diff
@@ -350,6 +350,36 @@ std::shared_ptr<Auth> YoutubeAuth::Login(QWidget *owner,
 }
 
 #ifdef BROWSER_AVAILABLE
+void YoutubeChatDock::timerEvent(QTimerEvent *event)
+{
+	if (event->timerId() != awaitLoginTimer)
+		return;
+
+	QPointer<YoutubeChatDock> this_ = this;
+	auto cb = [this_](bool found) {
+		bool previouslyLoggedIn = this_->isLoggedIn;
+		this_->isLoggedIn = found;
+		if (found ^ previouslyLoggedIn) {
```

Comment:
Please do not use binary comparators. They are terrible for code readability.

Reply:
Done.